web/assignments.go

```diff
@@ -38,10 +35,16 @@ func (s *AutograderService) updateAssignments(ctx context.Context, sc scm.SCM, c
 	if err != nil {
 		return err
 	}
-	assignments, err := assignments.FetchAssignments(ctx, sc, course)
+	assignments, dockerfile, err := assignments.FetchAssignments(ctx, sc, course)
```

Comment:
In progress.

Reply:
Done: `UpdateFromtestRepo` could actually be used in both cases, there was no need for a separate function as `updateAssignments` since we do not really need to pass a repository to the former anymore.